SignalMessaging/categories/UIFont+OWS.m

```diff
@@ -72,14 +72,19 @@ + (UIFont *)ows_dynamicTypeHeadlineFont
     return [UIFont preferredFontForTextStyle:UIFontTextStyleHeadline];
 }
 
-+ (UIFont *)ows_dynamicTypeSubheadlineFont
++ (UIFont *)ows_dynamicTypeBodyFont
 {
-    return [UIFont preferredFontForTextStyle:UIFontTextStyleSubheadline];
+    return [UIFont preferredFontForTextStyle:UIFontTextStyleBody];
 }
 
-+ (UIFont *)ows_dynamicTypeBodyFont
++ (UIFont *)ows_dynamicTypeCalloutFont
```

Comment:
No significant changes to this file - I mistakenly thought I needed to use this font, but as long as it's available, we might as well include it. 

I also tweaked the ordering to be monotonically decreasing in line with the HIG.

Reply:
I thought the design was to only use four font styles?  When would we use "call out"?

Reply:
We don't currently use it anywhere. We have literally every other font from the hig in this class, including ones we don't use. After accidentally adding it I thought I'd keep it since we might use it someday, but I didn't think it would be controversial.

Reply:
Not controversial, just asking.